packages/manager/apps/web/client/app/hosting/cdn/shared/hosting-cdn-shared-settings.constants.js

```diff
@@ -0,0 +1,10 @@
+export const HOSTING_CDN_SHARED_SETTINGS_RULE_TYPE_DEVMODE = 'devmode';
+export const HOSTING_CDN_SHARED_SETTINGS_RULE_TYPE_BROTLI = 'brotli';
+export const HOSTING_CDN_SHARED_SETTINGS_RULE_CACHE_RULE = 'cache_rule';
+export const HOSTING_CDN_SHARED_SETTINGS_RULE_CACHE_RULE_PATTERN_REGEX =
+  'regex';
+export const HOSTING_CDN_SHARED_SETTINGS_RULE_CACHE_RULE_PATTERN_EXTENSION =
+  'extension';
+export const HOSTING_CDN_SHARED_SETTINGS_RULE_FACTOR_DAY = 86400;
+export const HOSTING_CDN_SHARED_SETTINGS_RULE_FACTOR_HOUR = 3600;
+export const HOSTING_CDN_SHARED_SETTINGS_RULE_FACTOR_MINUTE = 60;
```

Comment:
Maybe we can omit the `HOSTING_CDN_SHARED_SETTINGS_` prefix. When we will use it, we know that we import it from `client/app/hosting/cdn/shared/hosting-cdn-shared-settings.constants`.

Reply:
i changed it bit (i removed HOSTING_)